staging/src/k8s.io/apiserver/pkg/server/filters/goaway_test.go

```diff
@@ -305,3 +431,94 @@ func TestHTTP1Requests(t *testing.T) {
 		t.Errorf("expect response HTTP header Connection to be empty, but got: %s", v)
 	}
 }
+
+// TestGOAWAYConcurrency tests GOAWAY frame will not affect concurrency requests in a single http client instance.
+// Known issues in history: https://github.com/kubernetes/kubernetes/issues/91131.
+func TestGOAWAYConcurrency(t *testing.T) {
```

Comment:
this test fails 100% of the time when run with `go test -race k8s.io/apiserver/pkg/server/filters -run TestGOAWAYConcurrency`, disabling in https://github.com/kubernetes/kubernetes/pull/94529 and opened https://github.com/kubernetes/kubernetes/issues/94532 to track fixing the test or the component

Reply:
caused by golang http/http2 pkg, I have already opened an issue [#41234](https://github.com/golang/go/issues/41234) .